test/library/standard/Set/setCommTest.good

```diff
@@ -0,0 +1,100 @@
+T1
+adding
+loc=0
+init int
+loc=1
+init int
+loc=2
+init int
+loc=3
+init int
+loc=4
+init int
+loc=5
+init int
+loc=6
+init int
+loc=7
+init int
+loc=8
```

Comment:
can you do something to make this test output more understandable? it's kindof long and monolithic